### PR TITLE
Hotfix v1.0.11 - concattenated_standardise Snowflake & Bigquery(default)

### PR DESCRIPTION
**Pull request by bschlottfeldt**
The check for alias was not yet implemented in the Snowflake and default versions of the macro concattenated_standardise and multi_active_concattenated_standardise.
This PR includes this fixes and also replaces the single quote in the  CONCAT() of Snowflake that was generating a formatting problem in VS Code, with double quote.

### CODE REVIEW

**Review by tkirschke — APPROVED** [review]
IMO it looks good, but before merging, @obause should look over it aswell, because he works more with Snowflake than I do.

**Comment by bschlottfeldt** [comment]
I will wait for his approval as well. Thanks @tkirschke

**Review by obause — APPROVED** [review]
Works in snowflake

**Pull request merged**
